docs/dev/guidelines/server.rst

```diff
@@ -329,9 +329,11 @@ The table contains all annotations for the corresponding minimum role. Different
 +------------------+----------------------------------------+
 | USER             | @PreAuthorize("hasRole('USER')")       |
 +------------------+----------------------------------------+
-| ANONYMOUS        | @PreAuthorize("permitAll()")           |
+| ANONYMOUS        | @EnforceNothing                        |
 +------------------+----------------------------------------+
 
+If, for some reason, you need to deviate from this rule, use ``@ManualConfig``. Use this annotation only if absolutely necessary as it will exclude the endpoint from the automatic authorization tests.
```

Comment:
What rule are you referring to?

Reply:
I changed it to `these rules`. I hope that makes it clear that it refers to the previously mentioned rules.